lib/web_ui/test/hash_codes_test.dart

```diff
@@ -1,127 +0,0 @@
-// Copyright 2013 The Flutter Authors. All rights reserved.
```

Comment:
I would keep this file (with suitable `//ignore`s for the deprecation warnings and with the numbers adjusted as necessary). That will act as an effective test that the deprecated API still works.

Reply:
That's the thing, the new API (`Object.hash`)'s seed value changes when the system is rerun, so the same input will give different hash values at every restart (so to speak). So I can't set a constant adjusted value. You can test this in dartpad,dev; Run the below code, see the output, reload the page, run it again, the output will be a different number.

```dart
void main() {
  print(Object.hash(0.0, 0.0.));
}
```

I could maybe change the test in a way that computes the hash twice and tests it if they are the same, if that helps.

Reply:
cc @yjbanov

Reply:
Restored the test.